js/MVP/queries/brev.ts

```diff
@@ -10,6 +10,17 @@ export const useBrev = () => {
   return useQuery(BREV, fetchBrev);
 };
 
+export const useSvarPaInnkallelse = (uuid: string) => {
+  const queryClient = useQueryClient();
+  const postSvar = (uuid, svar: SvarRespons) => post(`${ISDIALOGMOTE_API_BASE_PATH}/${uuid}/respons`, svar);
+
+  return useMutation((svar: SvarRespons) => postSvar(uuid, svar), {
```

Comment:
Her burde vi oppdatere veltet ved mutation.

Reply:
Tror ikke det skal være nødvendig i teorien, siden man har spinner på innsending. Tenker at flyten bør være noe ala: 

1. Sender inn svar
2. Får loading-animasjon frem til svaret er sendt
3. Fremvise beskjed om at svaret ditt er sendt, eventuelt feil dersom feil.

Reply:
Tenker på om vi ønsker å oppdatere visningen med verdien sendt inn uten å refreshe/refetche

Reply:
Blir ikke visningen oppdatert umiddelbart? Siden den er knyttet til formen, ikke til staten fra backend?

Reply:
Aha, sant det. Hvis frontenden rendrer riktig etter post så trenger vi ikke å gjøre noe.

Reply:
Jeg er ikke helt med på hva dere mener, men jeg har prøvd å gjøre det som er beskrevet her: https://tkdodo.eu/blog/mastering-mutations-in-react-query#invalidation

Sånn jeg skjønner det vil React Query fetche brevet på nytt når mutation er ferdig, og da vil brevet rendres på nytt og vise hva du har svart (basert på det som kommer fra backend).

Reply:
Ja, du har rett: https://react-query.tanstack.com/guides/query-invalidation . Jeg trodde den bare satt dataen til stale.